commons-geometry-euclidean/src/main/java/org/apache/commons/geometry/euclidean/threed/Plane.java

```diff
@@ -442,14 +593,51 @@ public double getOffset(final Vector3D point) {
         return point.dot(w) + originOffset;
     }
 
-    /** Check if the instance has the same orientation as another hyperplane.
+    /**
+     * Check if the instance has the same orientation as another hyperplane.
+     * 
      * @param other other hyperplane to check against the instance
-     * @return true if the instance and the other hyperplane have
-     * the same orientation
+     * @return true if the instance and the other hyperplane have the same
+     *         orientation
      */
     @Override
     public boolean sameOrientationAs(final Hyperplane<Vector3D> other) {
         return (((Plane) other).w).dot(w) > 0.0;
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public String toString() {
+        return "Plane [originOffset=" + originOffset + ", projectedOrigin=" + projectedOrigin + ", u=" + u + ", v=" + v + ", w=" + w
+                + ", precision=" + precision + "]";
```

Comment:
I would trim down the info here and just print out the most useful bits, namely `origin`, `u`, `v`, and `w`. Also, use the name `origin` instead of `projectedOrigin`, since the former is the name of the property as exposed to the outside world (via `getOrigin()`).

Reply:
done